snuba/snuba_migrations/replays/0007_add_replay_type_column.py

```diff
@@ -0,0 +1,74 @@
+from typing import List, Sequence, Tuple
+
+from snuba.clickhouse.columns import Column, Enum
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+new_columns: Sequence[Tuple[Column[Modifiers], str]] = [
+    (
+        Column(
+            "replay_type",
+            Enum([("session", 1), ("error", 2)], Modifiers(nullable=True)),
+        ),
+        "replay_id",
+    )
+]
+
+
+class Migration(migration.ClickhouseNodeMigrationLegacy):
```

Comment:
We are moving away from the `ClickhouseNodeMigrationLegacy` migrations, I think there is one last PR to be merged to be able to start using the new `ClickhouseNodeMigration`, I think this would be a good opportunity to use the new implementation provided that we get that PR merged cc @dbanda

Reply:
The PR is merged. To use `ClickhouseNodeMigration`, implement `forwards_ops` and `backwards_ops`. The `_local` and `_dist` functions are deprecated. Your ops now go in a single sequence.

Reply:
You use the `target=` parameter to your ops to specify if they should be local or distributed.

Reply:
Looks great!  Love the new system.  Sounds like a good project for my next migration.  I have the generated SQL I need though.